src/follower.py

```diff
@@ -73,35 +73,60 @@ def stop(self):
 
     """
     Turn the wheels towards the last recognized object.
+    Pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
     """
-    Converts the camera's angle scale to the same scale as the wheels.
+    Follows the tag by panning camera towards the same direction as the wheels.
+    
+    If the direction of the object is more/less than the distance then set the camera 3 steps 
+    to the left or right. Otherwise reset the camera to 90 degrees or 0 steps. 
+    """
+    def pan_camera(self):
+        if self._turn_angle < self._distance:
+            self.turn_camera_left(3)
+        elif self._turn_angle > self._distance:
+            self.turn_camera_right(3)
+        else:
+            self.reset_camera() 
+
+
+    """
```

Comment:
Add a comment for each function. If you're going to describe the parameter, explain what a step is. Again, this is just as bad as SunFounder doing this. Someone looking at this would have no idea what a step is. I would change this to degrees and do the conversion stuff yourself until we rewrite this. Remove the parameter description. We can add them when we change our comments to follow more of a documentation style.